content/docs/command-reference/import-url.md

```diff
@@ -125,11 +125,15 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create `.dvc` file without actually downloading `url`. The data
-  hash is not calculated when this option is used, only the import metadata is
-  saved to the `.dvc` file. It can be useful to skip the download if the file or
-  directory already exists locally, for example, along with `dvc commit` to
-  store it in the cache and record its hash value in the `.dvc` file.
+- `--no-exec` - create the import `.dvc` file without actually downloading
+  `url`. The data hash is not calculated when this option is used, only the
+  import metadata is saved to the `.dvc` file. It can be useful to skip the
+  download if the file/directory already exists locally, for example, along with
+  `dvc commit` to store it in the cache and record its hash value in the `.dvc`
+  file.
+
+  This is useful if, for example, you need to build a project that will use
```

Comment:
you have repetition useful ... useful

Reply:
Hm yeah this sentence wasn't great, sorry... Updated in d3ed5536.

Reply:
it's repetitive with the last sentence of the previous paragraph, essentially it reads:

```
something. It is useful when A.

This is useful when B.
```

is it intentional?

and overall it got complicated again. Is there a way to merge A and B?

Reply:
True. I guess this is why we didn't have so much motivation in the --no-exec options before... I tried to merge it now but it's still a little long.

Maybe we should remove the part about "if the file/directory already exists locally"? It seems like a very edge case to download something manually and THEN register it with `import`.

Reply:
> Maybe we should remove the part about "if the file/directory already exists locally"

Yeah, I removed that to simplify.